packages/docs/src/components/content/SearchForm.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { ChangeEvent, FormEvent, useEffect, useState } from 'react';
+import { Button, TextInput } from '@cmsgov/design-system';
+import { getQueryParamValue } from '../../helpers/urlUtils';
+
+const SearchForm = (props) => {
+  const [inputText, setInputText] = useState('');
+
+  const handleSubmit = (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (inputText) {
+      window.open(`${location.origin}/search?query=${inputText}`, '_self');
```

Comment:
What's the benefit of calling `window.open` vs setting the `location.href`? Was curious about this and did [some research](https://stackoverflow.com/a/23394403) that shows one might be more reliable than the other. We also seem to be doing it a third way [elsewhere in our doc site](https://github.com/CMSgov/design-system/blob/forrest-docsite-search/packages/docs/src/helpers/urlUtils.ts#L74-L76).

Reply:
A simpler option, too, might be to have no state or effects and let the HTML form behavior run wild without preventing default. But maybe there's a reason we can't do that?